static/css/thing.css

```diff
@@ -63,6 +80,16 @@
   margin: 0.5rem 0;
 }
 
+/* Hint at interactive things */
+.color-light,
+.dimmable-light,
+.on-off-light > .thing-icon,
+.smart-plug > .thing-icon,
+.on-off-switch > .thing-icon,
+.level-bar-container {
+  cursor: pointer;
```

Comment:
I actually think only the "expand thing" bubble should have the pointer cursor because it's a hyperlink that navigates somewhere, whereas tapping the thing icon behaves more like a button. The pointer is usually for hyperlinks and I wouldn't expect a hyperlink to toggle some state and stay on the same page.

Reply:
Oh right, that is how buttons work :+1: